src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -85,6 +86,11 @@ export type MultiComboboxProps<T> = {
    * item will be added to the selection. If not, then this item won't be added
    * */
   validateAddition?: (userEnteredInput: string, existing: T[]) => boolean;
+
+  /**
+   *
```

Comment:
I'm not feeling strongly about the length thing. For me I'd rather we either render it or we don't. I don't feel strongly though, so I'll defer to whatever @vorillaz believes

Reply:
Same here, I would suggest passing this one as `prop`. Something like `clearAfter?: number`. When it's undefined we shall not show "clear all" option trigger. IMHO it makes sense from a UX perspective.
We want to clear all the options for Log type filtering, but we don't need this one when creating a rule as the action is more regulated, it's a singleton and we are forcing the user to think twice about the input.
On the other hand, it makes more sense to use this one for quick actions or when the UI is not so "responsive" as the filters mentioned above.